postgres-testing/src/main/java/org/folio/postgres/testing/PostgresTesterContainer.java

```diff
@@ -36,6 +38,8 @@ public class PostgresTesterContainer implements PostgresTester {
    */
   public static final int SIMULATED_ASYNC_REPLICATION_LAG_MILLISECONDS = 300;
 
+  private static final String IMAGE_NAME = getImageName(System.getenv());
```

Comment:
Would it maybe be more convenient to pass this as a system parameter rather than as an environment variable, or perhaps support both? If it is a system parameter then it is more easily passed to `-D` in the same command as `mvn test`.

Or is it that this will only be used in a "multiple module" integration style test which perhaps would argue for just using an env var.

Reply:
Seems fine thanks.